internal/services/springcloud/spring_cloud_build_deployment_resource.go

```diff
@@ -56,6 +56,20 @@ func resourceSpringCloudBuildDeployment() *pluginsdk.Resource {
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
 
+			"active": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
this should be `Default: true` rather than computed, else (with the way this is written) this'll default to false and then we won't show a diff if this changes and it's not defined

Reply:
The reason is `active` can be set by `spring_cloud_active_deployment` too, so it must be computed otherwise it's not backward compatible.

Reply:
which would highlight the diff, insofar as users can then use `ignore_changes` on this field here if required - meaning we need to ensure the update conditionally updates this field as required too

Reply:
Got it, I'll remove the computed. But I think it should default to `false`, because user may have multiple deployments, and if it defaults to `true`, the configuration will fail, and user must specify `active = false`, it becomes required though it's an optional property. WDYT?